js/calendar.js

```diff
@@ -454,8 +454,8 @@ if(!String.prototype.formatNum) {
 		start.setHours(time_start[0]);
 		start.setMinutes(time_start[1]);
 		var end = new Date(this.options.position.end.getTime());
-		end.setHours(time_end[0]);
-		end.setMinutes(time_end[1]);
+		//end.setHours(time_end[0]);
```

Comment:
yep,

I have commented out those lines because 'all day event' has not appeared
properly - I think there is problem with setting of the border of the of
period. After I commented it out - it seemed to start working correctly -
but you can verify this.





2015-06-08 7:41 GMT+02:00 Sergey Romanov <notifications@github.com>:

> In js/calendar.js
> <https://github.com/Serhioromano/bootstrap-calendar/pull/491#discussion_r31887991>
> :
>
> > @@ -454,8 +454,8 @@ if(!String.prototype.formatNum) {
> >  		start.setHours(time_start[0]);
> >  		start.setMinutes(time_start[1]);
> >  		var end = new Date(this.options.position.end.getTime());
> > -		end.setHours(time_end[0]);
> > -		end.setMinutes(time_end[1]);
> > +		//end.setHours(time_end[0]);
>
> Why these lines are commented?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Serhioromano/bootstrap-calendar/pull/491/files#r31887991>
> .
>